src/main/resources/templates/home.html

```diff
@@ -4,113 +4,48 @@
-        </div>
-      </div>
-    </div>
+  <div id="wrapper">
 
-    <details>
-      <summary>View register fingerprint</summary>
-      <div class="fingerprint-container">
-        <span class="fingerprint-explanation">
-          <a href="https://gdstechnology.blog.gov.uk/2015/10/13/guaranteeing-the-integrity-of-a-register/">
-            See how the fingerprint works
-          </a>
-        </span>
-<pre class="fingerprint">
-  <code>{
-    "tree_size": 9803348,
-    "timestamp": 1447421303202,
-    "sha256_root_hash": "JATHxRF5gczvNPP1S1WuhD8jSx2bl+WoTt8bIE3YKvU=",
-    "tree_head_signature":
-    "BAMARzBFAiEAkKM3aRUBKhShdCyrGLdd8lYBV52FLrwqjHa5/YuzK7ECIFTlRmNuKLqbVQv0QS8nq0pAUwgbilKOR5piBAIC8LpS"
-  }</code>
-</pre>
-      </div>
-    </details>
+    <div th:replace="main.html::phase"></div>
 
     <div class="grid-row">
-      <div class="column-two-thirds">
-        <h2>About registers</h2>
-        <p>A register is a list of data that comes from a reliable source and is the only list of its kind.</p>
-        <p>
-          Each register is made up of records. Each record is made up of ‘fields’ and ‘values’.
-          When any of the information in a register changes, a new entry is added to the record.
-          The most recent entry is shown as the current version of the record.
-          Older entries are also kept as part of the record to show when it was changed.
-        </p>
+
+      <main class="column-two-thirds" id="main" role="main">
+        <h1 class="heading-large" th:utext="${friendlyRegisterName}"></h1>
+        <th:block th:utext="${registerText}"></th:block>
         <p>
-          You can find
-          <a href="https://www.gov.uk/government/publications/registers/registers">more information about registers on GOV.UK</a>
-          and
-          <a href="https://register.register.gov.uk">view all registers</a>.
+          <a href="/records" class="button">View Register</a>
         </p>
+        <p class="text">You can also <a href="/download">download a copy</a> of the register.</p>
+        <p class="text">The <span th:utext="${friendlyRegisterName}"></span> contains <span th:text="${totalRecords}"></span> records and includes the following fields:</p>
+        <ul>
+          <li th:each="field : ${register.fields}" th:text="${field}"></li>
+        </ul>
+        <h3 class="heading-small">Using this regsiter</h3>
+        <p>Download a copy of this register. You'll need to reguallry uodate it to make sure you have the most current verion of the data.</p>
+        <p>If you're building a service, you can use this data to create elements for forms.</p>
+        <p>You might need to combine data from this register with data from another source to do this.</p>
+      </main>
+
+      <div class="column-third">
+        <aside class="govuk-related-items" role="complementary">
+          <h2 class="heading-medium">About this register</h2>
+          <th:block th:replace="main.html::attribution"></th:block>
```

Comment:
This is nice except that in the design the `Managed by` heading has the same heading-size as `Last updated`. Here it the first is a `heading-medium` and the second is a `heading-small`. I understand it's because we use `heading-medium` for all other pages but is there away we can achieve the correct size on both?